### PR TITLE
✨Increasing the Max String Size on the Controller Display

### PR DESCRIPTION
**Pull request by WillXuCodes**
#### Summary:
 VCS and RMS have had their controller max string sizes at 19 for a while, and PROS also having this wouldn't be too hard of a change. This was done by changing `CONTROLLER_MAX_COLS` under `src/devices/controller.c` to 19.

#### Motivation:
There was an issue opened by @djava requesting that we do this, but the discussion surround this died pretty suddenly so I decided to open a PR regarding this to restart the conversation

##### References (optional):
fixes https://github.com/purduesigbots/pros/issues/217

#### Test Plan:

- [x] Build and Upload to a robot with a 19 character display, see if it shows up.

### CODE REVIEW

**Comment by kunwarsahni01** [comment]
LGTM image

**Comment by nathan-moore** [comment]
You'll probably have to increase this line https://github.com/purduesigbots/pros/blob/337b4c5593e50a0a68368cf7567a101fbe0b48aa/src/devices/controller.c#L261

File: src/devices/controller.c [hunk 1]
```diff
@@ -259,7 +259,7 @@ int32_t controller_print(controller_id_e_t id, uint8_t line, uint8_t col, const
 }
 
 int32_t controller_clear_line(controller_id_e_t id, uint8_t line) {
-	static const char* clear = "               ";
+	static const char* clear = "                   ";
```
Comment (nathan-moore) [inline src/devices/controller.c:262]: If you want to be fancy, you could assert that the lengths match up.

**Comment by djava** [comment]
Seems worth a try at least putting it to like 50 and and just filling it with I's.
It's probably a VexOS limit at 19 if that's what RMS does but not necessarily

File: src/devices/controller.c (hunk 1 above)
Comment (WillXuCodes) [reply src/devices/controller.c:262]: like, making it so that it prints " " * Max string length?

**Comment by WillXuCodes** [comment]
Somebody with a controller could probably try that, but for pros itself the 19 limit seems reasonable given the small amount of space left with the 19 character word in the test case.

**Comment by kunwarsahni01** [comment]
I tried increasing it to 20, the max number of characters that the controller will display is 19.

File: src/devices/controller.c (hunk 1 above)
Comment (djava) [reply src/devices/controller.c:262]: Does `char*` clear = ' ' * CONTROLLER_MAX_COLS` not work in C?

Comment (WillXuCodes) [reply src/devices/controller.c:262]: I don't think so... you'd probably have to use a for loop for it.

Comment (nathan-moore) [reply src/devices/controller.c:262]: strings aren't first class citizens in C. While what you have will do something, it won't do what you expect unless you want to know what 32 * 19 is.

Comment (nathan-moore) [reply src/devices/controller.c:262]:
```c
kassert(strlen(clear) == CONTROLLER_MAX_COLS)
```
Is what I had in mind

**Pull request merged**